app/lib/package/name_tracker.dart

```diff
@@ -153,15 +160,16 @@ class _NameTrackerUpdater {
   }
 
   Future _scan() async {
+    final now = DateTime.now().toUtc();
     final query = _db.query<Package>()..order('created');
     if (_lastTs != null) {
       query.filter('created >', _lastTs);
     }
     await for (Package p in query.run()) {
       if (_stopped) return;
       nameTracker.add(p.name);
-      _lastTs = p.created;
     }
+    _lastTs = now.subtract(const Duration(minutes: 5));
```

Comment:
Nice touch... 5 minutes should be safe... maybe add a comment that this is to account for cases where the indexing in datastore is lagging.

Actually, maybe it's safer to make this 15 min... or even 1 hour... As indexing could be behind a bit... also machines could have clock drift... and the number of packages uploaded in an hour is going to be small regardless.